economy/achievements.md

```diff
@@ -0,0 +1,12 @@
+# achievements
+achievements are goals that when completed, give you some rewards. 
+
+## how can i see achievements?
+you can use the ` $achievements ` command to view your unlocked achievements and your progress on them. if you want to know about a particular achievement, you can use:
+```
+$achievements view < name of achievement >
+```
+this will show the details of the achievement as well as the rewards you get for completing them. there are currently 34 unique qchievements in nypsi with each having 5 levels amounting to a total of 170 achievements. you can get a list of all achievements using `$achievements all`.
```

Comment:
for achievements specifically i'd ask you tell people to use the slash (/) command versions. it's easier for achievements, especially the find view command